app/controllers/cloud_network_controller.rb

```diff
@@ -223,17 +223,12 @@ def form_params
     # Admin_state_Up is true by default
     params[:enabled] = false unless params[:enabled]
 
-    options[:name] = params[:name] if params[:name]
-    options[:ems_id] = params[:ems_id] if params[:ems_id]
+    copy_params_if_present(options, params, %i[name ems_id qos_policy_id provider_network_type provider_physical_network provider_segmentation_id])
     options[:admin_state_up] = switch_to_bol(params[:enabled])
     options[:shared] = switch_to_bol(params[:shared])
     options[:external_facing] = switch_to_bol(params[:external_facing])
     # TODO: uncomment once form contains this field
     # options[:port_security_enabled] = params[:port_security_enabled] if params[:port_security_enabled]
```

Comment:
I feel like you can also remove the commented lines, as they are 3 years old now